webhook/core/pod_webhook_test.go

```diff
@@ -153,6 +192,48 @@ func TestPodResourceInjector_Handle(t *testing.T) {
 	}
 }
 
+// TestFargatePodResourceInjector_Handle test webhook annotating fargate pod with sg list.
+func TestFargatePodResourceInjector_Handle(t *testing.T) {
```

Comment:
This is not necessary for this PR. But I think we should start using table driven tests to avoid repeated codes and improve readability and cases coverage. when we add or rewrite tests in this file. I think we have some in this project already.
https://github.com/golang/go/wiki/TableDrivenTests